app/views/accessibility/show.html.erb

```diff
@@ -0,0 +1,94 @@
+<%= content_for :title, page_title(:accessibility) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl"><%= t('page_titles.accessibility') %></h1>
+
+    <p class="govuk-body">
+      Apply for teacher training is a pilot service run by the
+      Department for Education. It allows people to apply to some teacher
+      training programmes in the UK.
+    </p>
+    <p class="govuk-body">
+      This service is designed to be used by as many people as possible.
+      We have built this service in line with the <%= govuk_link_to 'accessible design principles used across GOV.UK', 'https://design-system.service.gov.uk/accessibility/', target: '_blank' %>.
+    </p>
+    <p class="govuk-body">
+      The service is being piloted with a limited number of users. We have tested
+      the prototype with users with accessibility needs, for instance, users who
+      require screen readers.
+    </p>
+    <p class="govuk-body">
+      We will carry out a full accessibility audit after or during the pilot.
+    </p>
+    <p class="govuk-body">
+      <%= govuk_link_to 'AbilityNet', 'https://mcmw.abilitynet.org.uk/', target: '_blank' %>
+      has advice on making your device easier to use if you have a disability.
+    </p>
+
+    <h2 class="govuk-heading-l">
+      What to do if you cannot access parts of this website
+    </h2>
+    <p class="govuk-body">
+      If you need information on this website in a different format like accessible
+      PDF, large print, easy read, audio recording or braille:
+    </p>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>email <%= bat_contact_mail_to %></li>
+    </ul>
```

Comment:
![Screenshot 2019-10-18 at 12 44 52](https://user-images.githubusercontent.com/1650875/67091661-25590680-f1a5-11e9-86c9-87ad655811f7.png)

This `<ul>` probably can be inlined

Reply:
Updated in latest changes to this content.